downstream/assemblies/platform/assembly-disconnected-installation.adoc

```diff
@@ -13,7 +13,7 @@ If you are not connected to the internet or do not have access to online reposit
 
 Before installing {PlatformNameShort} on a disconnected network, you must meet the following prerequisites:
 
-. A created subscription manifest. See link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_operations_guide/assembly-aap-obtain-manifest-files#doc-wrapper[Obtaining a manifest file] for more information.
+. A subscription manifest that you can upload to the platform. See link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_operations_guide/assembly-aap-obtain-manifest-files#doc-wrapper[Obtaining a manifest file] for more information.
```

Comment:
FYI that this is going to be moved to the new Access management and authentication guide. Aine also set up attributes for the book URLs to make this easier going forward. So the link should be:
{URLCentralAuth}/index#assembly-aap-obtain-manifest-files

Reply:
Change made.